include/rdma/fi_log.h

```diff
@@ -35,6 +35,8 @@
 #ifndef _FI_LOG_H_
 #define _FI_LOG_H_
 
+#include "config.h"
```

Comment:
> 1. That stanza is actually incorrect, it should be #ifdef instead of #if

The source code needs updating then.

> 2. We always invoke AC_CONFIG_HEADERS in configure.ac, which means that we
> will always be compiled with -DHAVE_CONFIG_H
> <http://www.gnu.org/software/autoconf/manual/autoconf.html#Configuration-
> Headers>  and there's not any case where we should ever be missing
> config.h.

There's got to be a reason why this check is in all of the other OFA code.  Maybe custom build scripts?